man/systemd.unit.xml

```diff
@@ -1658,6 +1658,30 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ConditionMemoryPressure=</varname></term>
+          <term><varname>ConditionCPUPressure=</varname></term>
+
+          <listitem><para>Verify that the overall system (memory or CPU) pressure is below the given threshold.
+          This setting takes a colon-separated tuple as argument. The first value is the slice unit under which
+          the pressure will be checked, and it is optional: if not specified, the overall system pressure will be
+          measured, instead of a particular cgroup's.</para>
+
+          <para>The second value is the threshold. It can be specified as a simple percentage value, suffixed with
+          <literal>%</literal>, in which case the pressure will be measured as an average over the last five
+          minutes before the attempt to start the unit is performed, and it will be checked if it is strictly
+          below the specified threshold.</para>
+
+          <para>Alternatively, the average type and the comparison operator can also be specified, for example:
+          <literal>avg10 &gt;= 10%</literal>. The supported averages follow what the kernel provides, and are
+          <literal>avg10</literal>, <literal>avg60</literal> and <literal>avg300</literal>. The
```

Comment:
i am not a fan of exposing avg300 here too much, it's too low-level for my taste. Maybe use our usual way to write times instead, and then a suffix notation?

i.e. 

```
ConditionMemoryPressure=90%/5min
```

Reply:
Given I point to the kernel.org docs I kept the same notation for consistency, but I can change it

Reply:
please do. I really think we should normalize our interfaces to be internally consistent with our own, and put less focus on directly mapping the underlying kernel interfaces.